R-package/tests/testthat/test_lgb.Booster.R

```diff
@@ -1,3 +1,27 @@
+context("Boooster")
```

Comment:
```suggestion
context("Booster")
```

Reply:
I really need to get this keyboard cleaned haha